index.php

```diff
@@ -49,10 +49,18 @@
 $recent         = $robot->get_processed();
 $oldqueuesize   = $robot->get_old_queue_size();
 
-$broken = $DB->get_field_sql("SELECT COUNT(*)
+$numurlsbroken = $DB->get_field_sql("SELECT COUNT(*)
                                  FROM {linkchecker_url}
                                 WHERE httpcode != ?", array('200') );
 
+$sqlnumpages = "SELECT COUNT (*)
+                         FROM {linkchecker_url}  b
+                    LEFT JOIN {linkchecker_edge} l ON l.b = b.id
+                    LEFT JOIN {linkchecker_url}  a ON l.a = a.id
+                    LEFT JOIN {course} c ON c.id = a.courseid
+                        WHERE b.httpcode != '200'";
```

Comment:
This whole sql doesn't look right, and definitely isn't producing correct numbers.

Off the top of my head: (please test yourself)

```
SELECT COUNT (*)
  FROM {linkchecker_url}  b
  JOIN {linkchecker_edge} l ON l.b = b.id
 WHERE b.httpcode != '200'";
```